pkg/app/console.go

```diff
@@ -33,7 +38,16 @@ func (e *ConsoleService) GetGRPCRegistrations(services ...string) builder.GRPCRe
 
 func (e *ConsoleService) GetGatewayRegistration(services ...string) builder.HandlerRegistrations {
 	return func(ctx context.Context, mux *runtime.ServeMux, grpcEndpoint string, opts []grpc.DialOption) error {
-		return nil
+		return mux.HandlePath("GET", "/", runtime.HandlerFunc(func(w http.ResponseWriter, r *http.Request, pathParams map[string]string) {
+			var httpType string
+			if e.cfg.Gateway.HTTP {
+				httpType = "http"
+			} else {
+				httpType = "https"
+			}
+			redirectURL := fmt.Sprintf("%s://%s/ui/directory/model", httpType, e.cfg.Gateway.ListenAddress)
```

Comment:
This should be a relative redirect rather than an absolute.  We have no idea what hostname users are going to use, they might choose an alias or use a reverse proxy in front of topaz.

Reply:
Changed to use a relative path.